test/e2e/framework/volume/fixtures.go

```diff
@@ -297,8 +293,6 @@ func startVolumeServer(client clientset.Interface, config TestConfig) *v1.Pod {
 			},
 			Volumes:       volumes,
 			RestartPolicy: restartPolicy,
-			NodeName:      config.ServerNodeName,
```

Comment:
the only place where `ServerNodeName` was used seems to be here and it's just deleted and not replaced by anything 🤔

Reply:
correct, nothing is using it, so I just removed it.